specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json

```diff
@@ -1197,6 +1197,18 @@
             "impactType": {
               "type": "string",
               "description": "The type of the impact"
+            },
+            "maintenanceId": {
+              "type": "string",
+              "description": "Maintenance Id"
```

Comment:
Put better description so external users can understand the use case.
Something like
"This is a unique id for Planned Maintenance events used by some of our systems. "